server/tests/src/test/java/org/infinispan/server/functional/XSiteHotRodCacheOperations.java

```diff
@@ -1,54 +1,110 @@
 package org.infinispan.server.functional;
 
+import static org.infinispan.server.functional.XSiteIT.LON;
+import static org.infinispan.server.functional.XSiteIT.LON_CACHE_CUSTOM_NAME_XML_CONFIG;
+import static org.infinispan.server.functional.XSiteIT.LON_CACHE_OFF_HEAP;
+import static org.infinispan.server.functional.XSiteIT.NYC;
+import static org.infinispan.server.functional.XSiteIT.NYC_CACHE_CUSTOM_NAME_XML_CONFIG;
+import static org.infinispan.server.security.Common.sync;
+import static org.junit.Assert.assertEquals;
+
+import java.util.stream.IntStream;
+
 import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.rest.RestCacheClient;
+import org.infinispan.client.rest.RestClient;
 import org.infinispan.commons.configuration.XMLStringConfiguration;
-import org.infinispan.commons.test.Eventually;
-import org.infinispan.server.test.junit4.InfinispanServerRule;
-import org.infinispan.server.test.junit4.InfinispanServerTestMethodRule;
+import org.infinispan.commons.dataconversion.internal.Json;
+import org.infinispan.server.test.junit4.InfinispanXSiteServerRule;
+import org.infinispan.server.test.junit4.InfinispanXSiteServerTestMethodRule;
 import org.junit.ClassRule;
 import org.junit.Rule;
 import org.junit.Test;
 
 /**
  * @author Pedro Ruivo
+ * @author Gustavo Lira
  * @since 11.0
  **/
 public class XSiteHotRodCacheOperations {
 
    @ClassRule
-   public static final InfinispanServerRule LON_SERVERS = XSiteIT.LON_SERVERS;
-   @ClassRule
-   public static final InfinispanServerRule NYC_SERVERS = XSiteIT.NYC_SERVERS;
-
-   private static final String LON_CACHE_XML_CONFIG =
-         "<infinispan><cache-container>" +
-               "  <distributed-cache-configuration name=\"%s\">" +
-               "     <backups>" +
-               "        <backup site=\"NYC\" strategy=\"ASYNC\"/>" +
-               "     </backups>" +
-               "  </distributed-cache-configuration>" +
-               "</cache-container></infinispan>";
-   @Rule
-   public InfinispanServerTestMethodRule LON_SERVER_TEST = new InfinispanServerTestMethodRule(XSiteIT.LON_SERVERS);
+   public static final InfinispanXSiteServerRule SERVERS = XSiteIT.SERVERS;
 
    @Rule
-   public InfinispanServerTestMethodRule NYC_SERVER_TEST = new InfinispanServerTestMethodRule(XSiteIT.NYC_SERVERS);
+   public InfinispanXSiteServerTestMethodRule SERVER_TEST = new InfinispanXSiteServerTestMethodRule(SERVERS);
+
+//   @Before
```

Comment:
should we remove the code commented?

Reply:
@diegolovison fixed, thanks for catch that